components/form/Form.tsx

```diff
@@ -180,8 +180,8 @@ export type WrappedFormUtils<V = any> = {
   /** 重置一组输入控件的值与状态，如不传入参数，则重置所有组件 */
   resetFields(names?: Array<string>): void;
   // tslint:disable-next-line:max-line-length
-  getFieldDecorator<T extends Object = {}>(
-    id: keyof T,
+  getFieldDecorator(
+    id: keyof V,
```

Comment:
Don't remember why we use a generic type `T` here, but we can't use `keyof V`, because of a field key can be a nested key path.

Reply:
@yesmeck can you describe what you mean by nested key path? Better yet, can you provide a snippet of what that object might look like?

Because I might be able to produce an even smarter type.

Reply:
We can write `getFieldDecorator('foo.bar')` which produces a `{ foo: { bar: '' } }` value.